mne/viz/_3d.py

```diff
@@ -320,8 +321,9 @@ def plot_trans(info, trans='auto', subject=None, subjects_dir=None,
         show EEG sensors in their digitized locations or projected onto the
         scalp, or a list of these options including ``[]`` (equivalent of
         False).
-    dig : bool
-        If True, plot the digitization points.
+    dig : bool | 'fiducials'
+        If True, plot the digitization points. "fiducials" to plot fiducial
+        points only.
```

Comment:
I think it would be `Use "fiducials" to ...` since fiducials are separate from `mri_fiducials`